plugin.php

```diff
@@ -22,7 +22,7 @@
 # the file exists in case someone's using Composer to load their dependencies in
 # a different directory.
 
-if ( file_exists( __DIR__ . '/vendor/autoload.php' ) ) {
+if ( ! class_exists( Plugin::class ) && is_file( __DIR__ . '/vendor/autoload.php' ) ) {
```

Comment:
- we spare a filesystem operation
- and refuse directories named `autoload.php`